### ISSUE TITLE
is there oauth2 example?

### ISSUE THREAD

**Issue by geminiyellow** [Microsoft/reactxp#714]
use react-native's or something others? 
any idea?

**Comment by erictraut** [comment]
There's no current example. What service are you looking to auth into? Some services (like Microsoft's AAD and MSA identity services) provide native SDKs. Other services are purely web-based, so integrating them into a native app requires the use of a WebView.

**Comment by sm1th** [comment]
Hi, we're building a ReactXP app and need to provide sign in to Azure AD B2C. Is there an existing ReactXP extension that we can use to achieve this, or will we have to write our own?

**Comment by deregtd** [comment]
Nothing existing.  I could release my super-crappy one.  Lemme see how easy it would be to package it up.

**Comment by deregtd** [comment]
Okay, that took longer than I expected, but it was probably a good exercise.  Here's the release:
https://github.com/deregtd/reactxp-msaadalauth

Feel free to submit bugs or patches for better docs/sample code, it'll help apply to my Microsoft project(s) using it. :)

**Comment by sm1th** [comment]
@deregtd Does this work with AD **B2C**? As far as I understood ADAL doesn't support B2C, but MSAL does (according to https://stackoverflow.com/a/40053341/1687469)

**Comment by deregtd** [comment]
No idea.  I thought B2C was a subset of ADAL standard support, but I've never looked into it at all.  Unfortunately I can't use MSAL inside Microsoft, so I'm stuck with ADAL for another year or two until they improve the system.

**Comment by sm1th** [comment]
Yeah I appreciate your time & efforts but I don't think this will work for B2C. 
Cheers
